core/src/test/scala/unit/kafka/tools/ConsoleConsumerTest.scala

```diff
@@ -128,12 +128,28 @@ class ConsoleConsumerTest extends JUnitSuite {
     val config = new ConsoleConsumer.ConsumerConfig(args)
 
     //Then
-    assertTrue(config.useNewConsumer)
+    assertFalse(config.useOldConsumer)
```

Comment:
Doesn't the old assertion work any more? We should probably keep it to ensure that we are compatible.